### PR TITLE
Feature/dashboard layout

### PR DESCRIPTION
**Pull request by Edds390**
Just some minor layout changes.

Features

- Legend table: A table with a header encloses the legend.

- The data table is now beneath the master chart.

_____

ISSUES

- If there are too many node to data source combinations the legend gets way too large. We can't solve with a scrollbar either as the legend requires you to hover over the graph.

### CODE REVIEW

File: src/components/GraphTutorialPopover/GraphTutorialPopover.js [hunk 1]
```diff
@@ -17,7 +20,7 @@ export default class PopoverExampleAnimation extends React.Component {
   handleClick = (event) => {
     // This prevents ghost click.
     event.preventDefault();
-
+    console.log(event.currentTarget);
```
Comment (NathanHenderson) [inline src/components/GraphTutorialPopover/GraphTutorialPopover.js]: Remove logs for production

File: src/components/DataTable/DropDownNodeMenu/DropDownNodeMenu.js [hunk 2]
```diff
@@ -53,7 +46,7 @@ export default class DropDownNodeMenu extends React.Component {
         <DropDownMenu
           value={this.state.value}
           onChange={this.handleChange}
-          style={styles.customWidth}
+          className="custom-width"
```
Comment (NathanHenderson) [inline src/components/DataTable/DropDownNodeMenu/DropDownNodeMenu.js:49]: Could the name custom-width be more descriptive

File: src/components/pages/DashboardPage/DashboardPage.css [hunk 3]
```diff
@@ -0,0 +1,6 @@
+.dashboard-page {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  width: 100%;
```
Comment (KenneyC) [inline src/components/pages/DashboardPage/DashboardPage.css:5]: Maybe subjected to change if the left panel is separate from the dashboard for graph info

**Review by KenneyC — APPROVED** [review]
Looks good to me 👍

File: src/components/pages/DashboardPage/DashboardPage.css (hunk 3 above)
Comment (Edds390) [reply src/components/pages/DashboardPage/DashboardPage.css:5]: We'll come back and change it once we have all the components on the page; it's hard to calculate the width percentages without all the components available.

**Review by SeoyoungC — APPROVED** [review]
Looks great! There does seem to be a small issue where hovering over a node causes the legend to extend down the screen. This causes the scrollbar to appear suddenly on the side of the browser, which shifts a few elements to the left. May potentially be an issue related to a browser/OS.
However, it is not an urgent problem and can be addressed later!

**Pull request merged**